specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2024-11-01/CertificateOrdersDiagnostics.json

```diff
@@ -0,0 +1,190 @@
+            "description": "The certificate order name for which the response is needed.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/DetectorResponseCollection"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List app service certificate detector response": {
+            "$ref": "./examples/Diagnostics_ListAppServiceCertificateOrderDetectorResponse.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CertificateRegistration/certificateOrders/{certificateOrderName}/detectors/{detectorName}": {
+      "get": {
+        "tags": [
+          "CertificateOrdersDiagnostics"
+        ],
+        "summary": "Microsoft.CertificateRegistration call to get a detector response from App Lens.",
+        "description": "Description for Microsoft.CertificateRegistration call to get a detector response from App Lens.",
+        "operationId": "CertificateOrdersDiagnostics_GetAppServiceCertificateOrderDetectorResponse",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "certificateOrderName",
+            "in": "path",
+            "description": "The certificate order name for which the response is needed.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "detectorName",
+            "in": "path",
+            "description": "The detector name which needs to be run.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "startTime",
+            "in": "query",
+            "description": "The start time for detector response.",
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "name": "endTime",
+            "in": "query",
+            "description": "The end time for the detector response.",
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "name": "timeGrain",
+            "in": "query",
+            "description": "The time grain for the detector response.",
```

Comment:
>time grain [](http://example.com/codeflow?start=32&length=10)

Consider explaining this in more detail. Is this a timespan that specifies a sample interval?